tracing-subscriber/src/fmt/mod.rs

```diff
@@ -941,22 +774,26 @@ mod test {
 
     #[test]
     fn impls() {
-        let f = format::Format::default().with_timer(time::Uptime::default());
-        let subscriber = Subscriber::builder().on_event(f).finish();
+        // let f = format::Format::default().with_timer(time::Uptime::default());
+        let f = Format::default().with_timer(time::Uptime::default());
+        let fmt = FmtLayer::builder().event_format(f).finish();
+        let subscriber = fmt.with_subscriber(Registry::default());
         let _dispatch = Dispatch::new(subscriber);
 
         let f = format::Format::default();
-        let subscriber = Subscriber::builder().on_event(f).finish();
+        let fmt = FmtLayer::builder().event_format(f).finish();
+        let subscriber = fmt.with_subscriber(Registry::default());
         let _dispatch = Dispatch::new(subscriber);
 
         let f = format::Format::default().compact();
-        let subscriber = Subscriber::builder().on_event(f).finish();
+        let fmt = FmtLayer::builder().event_format(f).finish();
+        let subscriber = fmt.with_subscriber(Registry::default());
         let _dispatch = Dispatch::new(subscriber);
     }
 
     #[test]
     fn subscriber_downcasts() {
-        let subscriber = Subscriber::new();
+        let subscriber = Subscriber::builder().finish();
```

Comment:
seems like an unnecessary change?
```suggestion
        let subscriber = Subscriber::new();
```